src/pysdmx/model/message.py

```diff
@@ -30,3 +35,24 @@ class ActionType(Enum):
     Replace = "replace"
     Delete = "delete"
     Information = "information"
+
+
+class Header(Struct, frozen=True, kw_only=True):
+    """Header for the SDMX-ML file."""
+
+    id: str = "test"
+    test: str = "true"
+    prepared: datetime = datetime.strptime("2021-01-01", "%Y-%m-%d")
+    sender: str = "Unknown"
```

Comment:
There is a convention to use ZZZ whenever the sender is unknown. It is coming from the IMF initially, but it is fairly widespread, as far as I know.

Reply:
We could change it, but never saw this ZZZ sender. I will make it as the default value.